packages/components/src/toggle-group-control/toggle-group-control-option-base/styles.ts

```diff
@@ -102,12 +102,13 @@ const isIconStyles = ( {
 }: Pick< ToggleGroupControlProps, 'size' > ) => {
 	const iconButtonSizes = {
 		default: '30px',
-		'__unstable-large': '34px',
+		'__unstable-large': '32px',
 	};
 
 	return css`
 		color: ${ COLORS.gray[ 900 ] };
 		width: ${ iconButtonSizes[ size ] };
+		height: ${ iconButtonSizes[ size ] };
```

Comment:
More than a FYI, but I've recently discovered and I quite like it as a way to more clearly state that an element should be a square

```suggestion
		height: ${ iconButtonSizes[ size ] };
		aspect-ratio: 1;
```

Reply:
Ooh, I haven't had a chance to use this yet! Great suggestion 🙏